### PR TITLE
check if detJdV > 0

### LINKED ISSUE #16: Warn if det(J) <= 0

### PR DESCRIPTION
**Pull request by fredrikekre**
Fix #16 

Should maybe be an error and not just a warning. But then our tests fail since we just random nodes to reinit the fe_values.

### CODE REVIEW

File: src/fe_values.jl [hunk 1]
```diff
@@ -105,7 +105,8 @@ function reinit!{dim, T}(fe_v::FEValues{dim}, x::Vector{Vec{dim, T}})
         for j in 1:n_func_basefuncs
             fe_v.dNdx[i][j] = Jinv ⋅ fe_v.dNdξ[i][j]
         end
-        fe_v.detJdV[i] = det(fev_J) * w
+        detJdV = det(fev_J) * w
+        detJdV > 0.0 ? fe_v.detJdV[i] = detJdV : warn("detJdV is not positive: detJdV = $(detJdV)")
```
Comment (KristofferC) [inline src/fe_values.jl]: Gauss quadrature weights are not necessarily positive. Just do:
```jl
detJ = det(fev_J)
detJ < 0 && warn("blabla")
fe_v.detJdV[i] = detJ * w
```
?

**Comment by fredrikekre** [comment]
Should it be an `ArgumentError`? I guess the `x`-vectors giving a negative `detJ` is a wrong argument.

**Comment by KristofferC** [comment]
Arg error is fine in my opinion.

**Comment by fredrikekre** [comment]
Hmm... The fail on AppVeyor actually says `detJ < 0.0` for one case... Thought my minor perturbation of the the reference coords should be ok.

**Comment by KristofferC** [comment]
0.5 is not so minor.

**Comment by fredrikekre** [comment]
Wait what? Oh, I thought I changed it to 0.1

**Pull request merged**